packages/components/src/VirtualizedList/HeaderIcon/HeaderIcon.scss

```diff
@@ -4,5 +4,9 @@ $tc-list-header-icon-margin-top: 3px !default;
 	display: flex;
 	flex-grow: 1;
 	justify-content: center;
-	margin-top: $tc-list-header-icon-margin-top;
+	padding-top: $tc-list-header-icon-margin-top;
```

Comment:
Adding some meta-attributes shouldn't change the style.
And btw, the new `padding` declaration use a `margin` variable

Reply:
yes i change that